electron/src/menu/TrayHandler.test.main.ts

```diff
@@ -81,21 +81,30 @@ describe('TrayHandler', () => {
       });
 
       it('flashes the app frame when app is not in focus and you receive new messages', async () => {
-        const tray = new TrayHandler();
-        tray.initTray(TrayMock);
-
-        const appWindow = new BrowserWindow({
-          show: false,
-          useContentSize: true,
-        });
-
-        const flashFrameSpy = spy(appWindow, 'flashFrame');
-
-        await appWindow.loadURL('about:blank');
-        assert.strictEqual(appWindow.isFocused(), false);
-        tray.showUnreadCount(appWindow, 2);
-        assert.ok(flashFrameSpy.firstCall.calledWith(true));
-        flashFrameSpy.restore();
+        if (process.platform === 'darwin') {
```

Comment:
This test fails under a macOS darwin system, since the function is not called on darwin. I added the comment and adjusted the test case